core/src/i18n/en.yaml

```diff
@@ -615,6 +615,7 @@ cron-jobs:
   create-modal:
     day-of-month: Day of Month
     day-of-week: Day of Week
+    disable-sidecar: Disable sidecar
```

Comment:
Maybe remove that translation and use the one from `jobs` section? Common stuff for Jobs and CronJobs is here anyway (see `ContainerSection.js`, which uses only `jobs.` prefixed translation keys and is used for both Jobs and CJs).